bin/tx-prover/src/proxy/mod.rs

```diff
@@ -725,6 +773,9 @@ impl BackgroundService for LoadBalancerState {
                 // Update the worker list with healthy workers
                 *workers = healthy_workers;
 
+                // Update the worker count metric
+                WORKER_COUNT.set(workers.len() as i64);
+
```

Comment:
Would it not be easier to handle the `WORKER_UNHEALTHY` metric here? The number of unhealthy workers would be just the difference between the original number of workers and the number of healthy workers, right?

Reply:
> The number of unhealthy workers would be just the difference between the original number of workers and the number of healthy workers, right?

Yes.

> Would it not be easier to handle the WORKER_UNHEALTHY metric here?

I'm moving it here, here we already have some metrics so I might be cleaner to group them together.